code/game/objects/structures/signs/signs_interactive.dm

```diff
@@ -27,7 +27,8 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/calendar, 32)
 
 /**
  * List of delamination counter signs on the map.
- * Required as persistence subsystem loads after the ones present at mapload, and to reset to 0 upon explosion.
+ * Required as persistence subsystem loads after the ones present at mapload.
+ * For delams, resets to 0 upon explosion.
```

Comment:
why the extra detail?

Reply:
Mistake. Earlier on I had the two counters sitting up there and I changed the comment and didn't make it a perfect revert.